src/integration/java/org/springframework/data/couchbase/IntegrationTestNoShutdownApplicationConfig.java

```diff
@@ -0,0 +1,63 @@
+package org.springframework.data.couchbase;
+
+import java.util.Collections;
+import java.util.List;
+
+import com.couchbase.client.java.env.CouchbaseEnvironment;
+import com.couchbase.client.java.env.DefaultCouchbaseEnvironment;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.env.Environment;
+import org.springframework.data.couchbase.config.AbstractCouchbaseConfiguration;
+import org.springframework.data.couchbase.config.CouchbaseConfigurer;
+import org.springframework.data.couchbase.core.CouchbaseTemplate;
+import org.springframework.data.couchbase.core.WriteResultChecking;
+import org.springframework.data.couchbase.core.query.Consistency;
+import org.springframework.data.couchbase.repository.support.IndexManager;
+import org.springframework.data.util.Streamable;
+
+/**
+ * Configuration for testing no shutdown
+ *
+ * @author Subhashni Balakrishnan
+ */
+public class IntegrationTestNoShutdownApplicationConfig extends AbstractCouchbaseConfiguration {
+
+	@Autowired
```

Comment:
NIT: Replace hard tabs with spaces? In this file and `CouchbaseEnvironmentNoShutdownProxyTest`.